### PR TITLE
Fixes #20974 - Procedures for Katello-Service stop and start command

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

**Comment by iNecas** [comment]
As per face 2 face discussion, the solution to the ordering should be by explicitly mentioning the migration steps instead of relying on the searching by tags.

**Comment by kgaikwad** [comment]
@iNecas ,
I am trying to check these procedures on real sat_instance but I am not able to create box using `satellite-6.1.12-rhel7.box`.
It is giving me error on file unpacking. 
I will try again to download file freshly and will notify the result once it is done.

File: definitions/features/katello_service.rb [hunk 1]
```diff
@@ -7,16 +7,15 @@ def make_stop(spinner, options = {})
     services = find_services_for_only_filter(running_services, options)
     if services.empty?
       spinner.update 'No katello service running'
-      yield
+      yield if block_given?
     else
       begin
         filters = "--only #{services.join(',')}"
         spinner.update 'Stopping katello running services..'
```
Comment (iNecas) [inline definitions/features/katello_service.rb]: `Stopping katello services` would sound better.

**Comment by iNecas** [comment]
Tested and works well, needs rebase as it's conflicting with other PR + one comment on spinner message. Otherwise good to go

**Comment by kgaikwad** [comment]
@iNecas , 
 Done! Updated the PR.

**Comment by iNecas** [comment]
Thanks @kgaikwad

**Pull request merged**